spec/system/page_objects/pages/admin.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Pages
+    class Admin < PageObjects::Pages::Base
+      def visit_filtered_plugin_setting(filter)
+        visit("/admin/site_settings/category/plugins?filter=#{filter}")
```

Comment:
If we change `toggle_setting` to be more targeted as suggested, then this might make more sense to just filter the settings list to a specific plugin e.g. `/category/voting`?

Reply:
> e.g. /category/voting?

Not every plugin adds itself to the secondary menu/has a route though. This the two plugins I've been backfilling tests for doesn't show up.

Reply:
Ohh sorry I didn't realise this filter was showing all the plugin settings e.g. `plugin:discourse-upvotes`, disregard :)